Packs/EmailCommunication/ReleaseNotes/2_0_9.md

```diff
@@ -0,0 +1,21 @@
+
+#### Scripts
+
+##### SummarizeEmailThreads
+
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
+##### DisplayEmailHtmlThread
+
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
```

Comment:
```suggestion
Updated the Docker image to: *demisto/python3:3.10.11.58677*.
```

Reply:
@ShirleyDenkberg same here for the auto-generated format.